package.json

```diff
@@ -1,13 +1,7 @@
 {
   "name": "network-meter",
-  "productName": "Network Meter",
   "version": "0.1.0",
-  "description": "Graphical frontend for various network tests",
   "main": "index.js",
-  "repository": {
-      "type": "git",
-      "url" : "https://github.com/hellais/network-meter"
-  },
```

Comment:
I think we would like to keep "productName", "description" and "repository" above, no?